tasks/main.yml

```diff
@@ -132,6 +132,17 @@
   when:
     - deploydrupal_drush_deploy
 
+# Config Structure Check - This is primarily to catch configuration structure
+# changes from updating modules with Dependabot or missed configuration. This
+# works because the exit code for grep is non-zero when the supplied search
+# string is not found.
+- name: config structure check
+  shell: "{{ deploydrupal_drush_path }} config:export -y | grep '{{ deploydrupal_config_check_structure_string }}'"
```

Comment:
```
TASK [ansible-deploy-drupal : config structure check] **************************
Wednesday 16 September 2020  14:48:56 +0000 (0:00:16.871)       0:02:15.668 *** 
fatal: [localhost]: FAILED! => changed=true 
  cmd: /var/lib/tugboat/vendor/bin/drush config:export -y | grep 'active configuration is identical'
  delta: '0:00:02.325950'
  end: '2020-09-16 14:48:59.012984'
  msg: non-zero return code
  rc: 1
  start: '2020-09-16 14:48:56.687034'
  stderr: ' [notice] The active configuration is identical to the configuration in the export directory (../drupal-config/sync).'
  stderr_lines: <omitted>
  stdout: ''
  stdout_lines: <omitted>
```
What am I missing here? This should be finding the string, right?

Reply:
```
10:39:10-adamzimmermann~/www/chromatichq.com (gh-2557-config-check)$ lando drush cex -y | grep 'active configuration is identical'
 [notice] The active configuration is identical to the configuration in the export directory (../drupal-config/sync).
--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- 10:39:31
10:39:31-adamzimmermann~/www/chromatichq.com (gh-2557-config-check)$ echo $?
0
```
Tested it locally and it seems to work.

Reply:
Could it have anything to do with the fact the message is output on `stderr` instead of `stdout`? I know I would expect this to work the same way both locally and in Tugboat, but maybe it doesn’t?

Also, do we know for a fact that you’re running identical versions of drush both in Tugboat and locally?

Reply:
@ctorgalson is this behavior related to how I'm calling it in Ansible or due to some Ansible quirk that I'm not aware of?

Reply:
@agarzola I think that was it. Adding `2>&1` fixed it. That merges the output of STDERR to STDOUT from my understanding.

Now to make it fail again due to a contrived circumstance.